apps/web/src/services/iam/components/UserGroupTable.vue

```diff
@@ -0,0 +1,122 @@
+<script setup lang="ts">
+import { computed, reactive } from 'vue';
+
+import { makeDistinctValueHandler, makeEnumValueHandler } from '@cloudforet/core-lib/component-util/query-search';
+import { getApiQueryWithToolboxOptions } from '@cloudforet/core-lib/component-util/toolbox';
+import { ApiQueryHelper } from '@cloudforet/core-lib/space-connector/helper';
+import { PToolboxTable } from '@cloudforet/mirinae';
+
+import { useQueryTags } from '@/common/composables/query-tags';
+
+import UserGroupTableToolbox from '@/services/iam/components/UserGroupTableToolbox.vue';
+import { calculateTime } from '@/services/iam/composables/refined-table-data';
+import { USER_GROUP_SEARCH_HANDLERS, USER_GROUP_STATE } from '@/services/iam/constants/user-group-constant';
+import { useUserGroupPageStore } from '@/services/iam/store/user-group-page-store';
+import { useUserPageStore } from '@/services/iam/store/user-page-store';
+
+interface Props {
+    tableHeight: number;
+}
+
+const userGroupPageStore = useUserGroupPageStore();
+const userGroupPageState = userGroupPageStore.state;
+
+const userGroupListApiQueryHelper = new ApiQueryHelper()
+    .setPageStart(userGroupPageState.pageStart).setPageLimit(userGroupPageState.pageLimit)
+    .setSort('name', true);
+let userGroupListApiQuery = userGroupListApiQueryHelper.data;
+
+const { queryTags } = useQueryTags({ keyItemSets: USER_GROUP_SEARCH_HANDLERS.keyItemSets });
+
+const userPageStore = useUserPageStore();
+
+const tableState = reactive({
+    fields: computed(() => [
+        { name: 'user_id', label: 'User ID' },
+        { name: 'name', label: 'Name' },
+        { name: 'state', label: 'State' },
+        { name: 'MFA', lable: 'mfa' },
+        { name: 'role_id', label: 'Admin Role', sortKey: 'role_type' },
+        { name: 'tags', label: 'Tags', sortable: false },
+        { name: 'auth_type', label: 'Auth Type' },
+        { name: 'last_accessed_at', label: 'Last Activity' }]),
+    items: computed(() => userGroupPageState.users.map((user) => ({
+        ...user,
+        mfa: user?.mfa?.state === 'ENABLED' ? 'ON' : 'OFF',
+        last_accessed_at: calculateTime(user?.last_accessed_at, userPageStore.timezone),
+    }))),
+    valueHandlerMap: computed(() => {
+        const resourceType = 'identity.User';
+
+        return {
+            user_id: makeDistinctValueHandler(resourceType, 'user_id'),
+            name: makeDistinctValueHandler(resourceType, 'name', 'string', [{ k: 'name', v: '', o: 'not' }]),
+            state: makeEnumValueHandler(USER_GROUP_STATE),
+            email: makeDistinctValueHandler(resourceType, 'email'),
+            auth_type: makeDistinctValueHandler(resourceType, 'auth_type'),
+            last_accessed_at: makeDistinctValueHandler(resourceType, 'last_accessed_at', 'datetime'),
+            tags: makeDistinctValueHandler(resourceType, 'tags'),
+        };
+    }),
+});
+
+const props = withDefaults(defineProps<Props>(), {
+    tableHeight: 400,
+});
```

Comment:
nitpick: For props, we usually place them at the top of the file for consistency. It would be great if you could do the same to keep it consistent with other files.